containers/daap-docs/src/var/task/swagger.json

```diff
@@ -228,10 +228,119 @@
           }
         }
       }
+    },
+    "/data-product/{data-product-name}/table/{table-name}/schema": {
+      "post": {
+        "tags": [
+          "schema"
+        ],
+        "summary": "Register a schema (blue print of your table) for a new table in your Data Product that you want to upload to the Data Platform.",
+        "description": "Registers the first version of your Data Product (by submitting it's metadata)  and enables you to proceed with uploading data.",
```

Comment:
I think this description needs updating - this looks like the description of the /register endpoint?

Reply:
yep have updated thanks